src/browser/provider/built-in/phantomjs.js

```diff
@@ -0,0 +1,47 @@
+import { create as createPhantom } from 'phantom';
+
+export default {
+    phantom: null,
+
+    openedPages: {},
```

Comment:
> If listAvailableBrowserNames returns [], it means that provider doesn't support multiple aliases, isn't that sufficient?

It's implicit and not clear. Regarding method name, yet another variant: `listAvailableOptionalBrowserNames` + I believe we should have `hasOptionalBrowserNames` flag.